LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -109,7 +109,7 @@ class LockboxXCUITests: BaseTestCase {
     }
 
     func testEntriesSortAndSearch() {
-        let firstEntryRecentOrder = "bmo.com"
+        let firstEntryRecentOrder = "arncyvuzox.co.uk"
```

Comment:
The joys of a shared account with test data that can be deleted? Ha yikes, thank you 🙏

Reply:
😛